getting-started/setup-multi-node-advanced.md

```diff
@@ -0,0 +1,370 @@
+  servers.
+- A *certificate* that is signed by the CA. This contains the public
+  key and is used to verify signatures and can also be freely copied.
+- A *key* that contains the private key. This is used to sign messages
+  and should be kept secure on the instance where it is generated.
+
+For the access node a single *root certificate* can be shared among 
+users. However each user on the access node needs a unique *signed 
+certificate* and *key*.
+
+For each data node, the *root certificate* can be copied from the access 
+node, but a *signed certificate* and *key* must be generated.
+
+##### 1. Set up a Certification Authority
+
+A Certificate Authority (CA) is necessary as a trusted third party to 
+sign other certificates.  The _key_ of the CA is used to sign 
+Certificate Signing Requests (CSRs), and the _certificate_ of the CA is 
+what is used as a root certificate for other parties. Creating a new CA 
+is not necessary if there is already one available to be employed.  In 
+that case skip to the next step.
+
+>:WARNING: The Certificate Authority key is central to your security
+and needs to be treated with care. Ideally, the key should be
+generated and used on a secured machine dedicated for this
+purpose. Typically, the key should not leave the machine and you
+should even consider the risk of backups being compromised.
+
+First, generate a key and insert it into your `auth.key` file:
+
+```bash
+openssl genpkey -algorithm rsa -out auth.key
+```
+
+Generate a self-signed root certificate for the CA:
+```
+openssl req -new -key auth.key -days 3650 -out root.crt -x509
+
+You are about to be asked to enter information that will be incorporated
+into your certificate request.
+What you are about to enter is what is called a Distinguished Name or a DN.
+There are quite a few fields but you can leave some blank
+For some fields there will be a default value,
+If you enter '.', the field will be left blank.
+-----
+Country Name (2 letter code) [AU]:US
+State or Province Name (full name) [Some-State]:New York
+Locality Name (eg, city) []:New York
+Organization Name (eg, company) [Internet Widgits Pty Ltd]:Example Company Pty Ltd
+Organizational Unit Name (eg, section) []:
+Common Name (e.g. server FQDN or YOUR name) []:http://cert.example.com/
+Email Address []: 
+```
+
+`Common Name` is the (imaginary) location of the root
+certificate, which in theory would allow a user to download the
+certificate from a known location and verify that it is the correct
+one. In this case, the value is not checked but it is possible to have
+different policies on what is allowed as common name.
+
+Remember that the root certificate only contains public information and
+can be distributed freely between machines. It is actually more secure
+to distribute it widely so that anyone interested in using the root
+certificate can be more certain to get the right certificate if it
+is available at many locations.
```

Comment:
IMO, there's too much "education" about certificates going on here, which is not relevant to our main message. I think we should refer to more authoritative documentation elsewhere for people that want to know more about how certificates and CAs work. I suggest we skip this section.

Also, the stuff about "common name" is confusing. The common name is typically the server name of the organization (I guess this is kind of a location, but when saying it is a "location" it is easily confused with other fields that are actual locations, like "Locality Name"). However, I don't think the common name is particularly important for a self-signed root CA certificate.